docs/ansible/azure/install.rst

```diff
@@ -0,0 +1,204 @@
+.. highlight:: console
+
+.. _azureInstallation-ref:
+
+Install Azure CLI
+=================
+In this section, you will install the Azure CLI and configure it so that you can provision and monitor your Azure resources from the command line.
+
+Install Azure
+-------------
+1. Install the Azure CLI using ``pip``::
+
+   $ pip install azure-cli
+
+
+2. Verify that the version of the Azure CLI is 2.0 or higher::
+
+   $ az --version
+
+3. Install a Python DNS resolver using ``pip``::
+
+   $ pip install dnspython
+
+4. Install a REST client for Azure using ``pip``::
+
+   $ pip install msrestazure
+
+
+Create an Azure service principal
+---------------------------------
+1. Login to Azure::
+
+   $ az login
+
+   You will be prompted to open a Web browser, go to a Web site, and enter a code in the text box presented at that site.  Following that, your browser will be redirected to login the http://login.microsoftonline.com.  Once you login there, your ``az login`` command is complete and your shell session is authenticated with Azure.
+
+2. Create an Active Directory application::
+
+   $ az ad app create --password abc123 --display-name myApp --homepage myapp.acme.com --identifier-uris myapp.acme.com
+
+   Your expected output will be similar to the following::
+
+      {
+      "appId": "1111111-111-1111-111-111111111",
+      "appPermissions": null,
+      "availableToOtherTenants": false,
+      "displayName": "myApp",
+      "homepage": "myapp.acme.com",
+      "identifierUris": [
+        "myapp.acme.com"
+      ],
+      "objectId": "55555555-5555-5555-5555-555555555555",
+      "objectType": "Application",
+      "replyUrls": []
+      }
+
+3. Create a resource group in a location near you::
```

Comment:
Some of this might already be done. Adding something like "if it does not already exist" might be a good idea. Mentioning the list command could be helpful, too.